proto/raw-js-openapi/build.gradle

```diff
@@ -30,12 +30,22 @@ Docker.registerDockerTask(project, 'webpackSources') {
         from ('Dockerfile')
         from ('../package.json')
         from ('../package-lock.json')
-
     }
     parentContainers = [ Docker.registryTask(project, 'node') ]
     imageName = 'deephaven/js-out:local-build'
     containerOutPath = '/usr/src/app/raw-js-openapi/build/js-out'
     copyOut {
-        into(webpackSourcesLocation)
+        include 'dh-internal.js'
+        into(webpackSourcesLocation.get().dir('jsapi'))
+    }
+}
+
+configurations {
+    js
+}
+
+artifacts {
+    js(webpackSourcesLocation) {
+        builtBy webpackSources
```

Comment:
The return is the task itself (well, a provider of it) so that other tasks can depend on it. Some tasks (not Sync) can be used as the value of the artifact. Passing the same "output dir" seems like the cleanest way I've found for this so far. In this case, webpackSources is the task, which we still need, so we can tell gradle what built the artifact (this auto-wires up the assemble task, among other things).

Some iteration will be required - the docker helper that creates a py wheel also returns the task, but it needs to actually yield the .whl file itself so we don't accidentally get old files.